ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/exception/ForbiddenException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 ELIXIR EGA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package eu.elixir.ega.ebi.dataedge.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.FORBIDDEN)
+public class ForbiddenException extends RuntimeException {
+    public ForbiddenException(String message) {
+        super(message);
+    }
+}
```

Comment:
@anandmohan777 is this an Exception that you can handle with the exciting 
`PermissionDenied | 403 | Authorization is required to access the resource` that we also use in Htsget? I'm up to reuse as much code as we have however, have a look if it will work for you or not :) But I think conceptually they are the same.